src/spring-cloud/azext_spring_cloud/vendored_sdks/appplatform/v2022_01_01_preview/aio/operations/_build_service_operations.py

```diff
@@ -692,14 +692,14 @@ async def get_resource_upload_url(
         return deserialized
     get_resource_upload_url.metadata = {'url': '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AppPlatform/Spring/{serviceName}/buildServices/{buildServiceName}/getResourceUploadUrl'}  # type: ignore
 
-    async def list_supported_buildpacks(
+    async def list_supported_buildpack(
```

Comment:
What these changes in SDK? The SDK is generated from swagger. It will be covered when the SDK re-generated

Reply:
should be mistake by replacing all buildpacks, let me refine it.

Reply:
Updated.